src/games/strategy/triplea/ai/proAI/ProNonCombatMoveAI.java

```diff
@@ -1905,6 +1905,17 @@ public int compare(final ProTerritory t1, final ProTerritory t2) {
               continue;
             }
 
+            // Check if territory is safe after all current moves
+            final List<Unit> defendingUnits = moveMap.get(t).getAllDefenders();
+            moveMap.get(t).setBattleResult(
+                calc.calculateBattleResults(player, t, moveMap.get(t).getMaxEnemyUnits(), defendingUnits, moveMap
+                    .get(t).getMaxEnemyBombardUnits(), false));
+            final ProBattleResult result = moveMap.get(t).getBattleResult();
+            if (result.getWinPercentage() >= ProData.minWinPercentage || result.getTUVSwing() > 0) {
```

Comment:
This logic can be simplified/pushed into ProBattleResult, ie:
result.shouldHold( ProData.minWinPercentage ), since the ProData value is static, it can be a no-arg method call.